web.config

```diff
@@ -9,5 +9,8 @@
                   stdoutLogFile="%HOME%\LogFiles\stdout"
                   arguments="-Djava.net.preferIPv4Stack=true -Dserver.port=%HTTP_PLATFORM_PORT% -jar &quot;%HOME%\site\wwwroot\send-letter-producer-service.jar&quot;">
     </httpPlatform>
+    <applicationInitialization>
```

Comment:
This looks like simplest solution to get rid of those 404's in app insights 👍 .Do you want to test this branch first in sandbox because none of the project has this at the moment and I think it will be good to get this tested in cnp branch before we merge this master?

Reply:
😮 changed base branch. should we get whole things in or make separate pr?

Reply:
I think push this to cnp branch and then trigger the build from sandbox Jenkins and then you can validate app insights

Reply:
here we go then 🏎